eslint.config.cjs

```
@@ -13,10 +13,14 @@ const lodashPlugin = require('eslint-plugin-lodash');
 const mochaPlugin = require('eslint-plugin-mocha');
 const prettierPluginRecommended = require('eslint-plugin-prettier/recommended');
 const securityPlugin = require('eslint-plugin-security');
+const botbuilderPlugin = require('./tools/eslint/plugins/botbuilder');
 const path = require('path');
 
 module.exports = config = [
```

ESLint

[error] 'config' is not defined.

(no-undef)